draft-ietf-tls-tls13.md

```diff
@@ -2525,9 +2525,19 @@ extension type ("supported_groups" extension):
 
        00 0A 00 06 00 04 00 17 00 18
 
+
+As of TLS 1.3, servers are permitted to send the "supported_groups"
+extension to the client.  If the server has a group it prefers to the
+ones in the "key_share" extension but is still willing to accept the
+ClientHello, it SHOULD send "supported_groups" to update the client's
+view of its preferences.  Clients MUST NOT act upon any information
+found in "supported_groups" prior to successful completion of the
+handshake.
```

Comment:
...but they MAY use the information learned from a successfully completed handshake to change what groups they offer to a server in subsequent connections.